src/lib/plugin.js

```diff
@@ -97,13 +137,23 @@ class FiveBellsLedger extends EventEmitter2 {
   }
 
   * _connect () {
-    const accountUri = this.account
-
     if (this.connection) {
       debug('already connected, ignoring connection request')
       return Promise.resolve(null)
     }
 
+    if (this.identifier) {
+      const newOptions = yield webfingerToOptions(this.identifier)
+
+      this.credentials = Object.assign(
```

Comment:
`Object.assign` is destructive, so you usually want to pass an empty object as the first parameter. In this case, it's not causing any problems, but if somebody changes how `webfingerToOptions` works... suddenly you have a subtle bug. So I'd create a new object just to be safe.